### ISSUE TITLE
Custom matplotlib axis no longer a valid argument for plotCutQE

### ISSUE THREAD

**Issue by abehersan** [MJOLNIRPackage/MJOLNIR#73]
I believe this has to do with the added functionality of the created axes within MJOLNIR that renders externally generated axes incompatible when passed on as arguments to the plotting function `plotCutQE`.

Error traceback is as follows:

`
Traceback (most recent call last):
  File "/home/abe/Projects/PSI/Software/YbBr3_FieldDependence/camea_dispersion.py", line 27, in <module>
    ax_qq0_contour, _, _ = data.ybbr3_10T.plotCutQE(**qe_params)
  File "/home/abe/miniconda3/envs/workenv/lib/python3.10/site-packages/MJOLNIR/_tools.py", line 71, in newFunc
    returnval = func(*args,**kwargs)
  File "/home/abe/miniconda3/envs/workenv/lib/python3.10/site-packages/MJOLNIR/Data/DataSet.py", line 1025, in plotCutQE
    pos = ax.calculatePositionInv(HKL)
AttributeError: 'AxesSubplot' object has no attribute 'calculatePositionInv'
`

I tested custom axes for `plotCutQELine` and didn't run into the issue there so it might not be a general issue. Haven't tried 1D plot cut methods yet.

**Comment by abehersan** [comment]
```
(workenv) [abe@tx1c ~]$ pip show MJOLNIR
Name: MJOLNIR
Version: 1.2.1

```

**Comment by Jakob-Lass** [comment]
Hoi Abraham

Ich habe es hoffentlich gelöst und es hochgeladen in den Develop Branch.

LG,
Jakob

**Comment by Jakob-Lass** [comment]
Solved in 32c5c0fc4ad0f53f0811358a05368cbc918ae79b by checking for presence of distance function. If not found utilize standard distance.

**Comment by abehersan** [comment]
attempted fix in pull req. #74 - testing for `plotCut1D` pending
